src/services/queryServices.js

```diff
@@ -0,0 +1,40 @@
+import QueryModel from '../models/queryModel'
+
+export default class QueryServices {
+
+    async createQuery(data) {
+        try {
+            const query = new QueryModel(data);
+            await query.save();
+            return query;
+        } catch (error) {
+            throw error;
```

Comment:
Return appropriate message for all methods

Reply:
i implemented all my error message structuring in my controller, and as this is a service method it will just send the error to the controller to be structured and sent.

Reply:
i implemented all my error message structuring in my controller, and as this is a service method it will just send the error to the controller to be structured and sent to the user.